File: ruby/fibonacciRecursive.rb
# Fibonacci Recursive Algorithm in Ruby
#
# Author: Elton Fonseca
# 

def fibonacci(term)
    if term == 1 || term == 2
        return 1
    else
        return fibonacci(term - 1) + fibonacci(term - 2) 
    end
end

puts fibonacci(10)

File: javascript/fibonacciRecursive.js
/**
 *  Fibonacci Recursive in JavaScript
 * 
 * Author: Elton Fonseca
 */

function fibonacci(term)
{
    if(term == 1 || term == 2)
        return 1;
    else 
        return fibonacci(term - 1) + fibonacci(term - 2);
}

console.log(fibonacci(10));